.github/workflows/ci.yml

```
@@ -46,6 +50,7 @@ jobs:
       uses: actions/setup-node@39370e3970a6d050c480ffad4ff0ed4d3fdee5af # 4.1.0
       with:
         node-version: ${{ matrix.node.version }}
+        env: NODE_OPTIONS=${{ matrix.node.options }}
```

```suggestion
        env: NODE_OPTIONS="${{ matrix.node.options }}"
```
considering there's spaces and newlines in there?

---

I'm not sure if the quotes are necessary; as long as they don't hurt, sure? There are no new lines though thanks to the yaml magic of `>-` (which collapses inter-whitespace into a single space character).